File: Java/src/com/hackerrank/daysofcode/LetsReview.java
package com.hackerrank.daysofcode;
import java.io.*;
import java.util.*;
/**
 * Created by vmurakami on 23/07/17.
 */
public class LetsReview {
    public static void main(String[] args) {


        Scanner scan = new Scanner(System.in);
        int qty_words = scan.nextInt();
        while (qty_words > 0) {
            String word = scan.next();

            StringBuilder even = new StringBuilder();
            StringBuilder odd = new StringBuilder();

            for (int i = 0; i < word.length(); i++) {
                if (i % 2 == 0 || i == 0) {
                    even.append(word.charAt(i));
                } else {
                    odd.append(word.charAt(i));
                }
            }
            qty_words--;
            System.out.println(even.toString() + " " + odd.toString());
        }
    }
}

File: Scala/src/main/scala/LetsReview.scala
package main.scala

object LetsReview {
  def main(args: Array[String]) {
  val qty_word:Int = scala.io.StdIn.readInt()
    while(qty_word > 0){
      var word:String = scala.io.StdIn.readLine()
      var even = new StringBuilder()
      var odd = new StringBuilder()
      for (counter <- 0 to word.length - 1){
        if(counter % 2 == 0 || counter == 0){
          even.append(word.charAt(counter))
        } else {
          odd.append(word.charAt(counter))
        }

      }
      println(s"$even $odd")
      }
    }
}
